src/components/ProfilePage/ProfilePage.css

```diff
@@ -10,7 +10,6 @@
     width: 100%;
     max-width: 360px;
     background-color: white;
-    margin-top: 20px;
```

Comment:
If you're going to set 3-4 margins, you can do it with one line like 'margin: (top) (right) (bottom) (left);'  //  margin: 20px 0px 40px 50px.